### PR TITLE
add link to FAF

### PR DESCRIPTION
**Pull request by xsuchy**
FAF is Fedora Analysis Server which contains aggregated reports of tracebacks or core dumps

### CODE REVIEW

File: fedoracommunity/widgets/package/templates/package_chrome.mak [hunk 1]
```diff
@@ -54,10 +54,10 @@ icon_url = tg.url("/images/icons/%s" % icon)
...
-                   <li><a class="other-app" href="http://koji.fedoraproject.org/koji/search?match=glob&type=package&terms=${quote(w.package_info['name'])}"><img src = "https://fedoraproject.org/static/images/icons/fedora-infra-icon_koji.png"/> Koji Builds </a> </li>
                    <li><a class="other-app" href="https://bugzilla.redhat.com/buglist.cgi?component=${w.package_info['name']}&query_format=advanced&product=Fedora&bug_status=NEW&bug_status=ASSIGNED&bug_status=REOPENED"><img src = "https://admin.fedoraproject.org/community/images/16_bugs.png"/> Bugzilla </a> </li>
-                   <li><a class="other-app" href="https://pkgs.fedoraproject.org/cgit/${w.package_info['name']}.git"><img src = "https://apps.fedoraproject.org/img/icons/git-logo.png"/> SCM </a> </li>
+                   <li><a class="other-app" href="http://koji.fedoraproject.org/koji/search?match=glob&type=package&terms=${quote(w.package_info['name'])}"><img src = "https://fedoraproject.org/static/images/icons/fedora-infra-icon_koji.png"/> Koji Builds </a> </li>
```
Comment (bowlofeggs) [inline fedoracommunity/widgets/package/templates/package_chrome.mak]: This appears to do more than just sort alphabetically, as it changes the koji link from a TLS URL to a plaintext URL.

Comment (xsuchy) [inline fedoracommunity/widgets/package/templates/package_chrome.mak]: Are you sure? I think you are comparing link of pkgs (which has TLS) with koji (which has pure http). The original link of koji is on line 57 and does not have https.

**Comment by pypingou** [comment]
Looks fine to me :)

File: fedoracommunity/widgets/package/templates/package_chrome.mak (hunk 1 above)
Comment (bowlofeggs) [inline fedoracommunity/widgets/package/templates/package_chrome.mak]: @xsuchy Oh I see that I was mistaken, I now see the plaintext URL a few lines above. Sorry for the confusion!

**Comment by pypingou** [comment]
@bowlofeggs ok to merge for you as well?

**Comment by bowlofeggs** [comment]
@pypingou Sure, LGTM!

**Comment by pypingou** [comment]
Let's merge then :)

**Pull request merged**